### PR TITLE
kronicways-ServerItemDefinitions-changes

### PR DESCRIPTION
**Pull request by QuentinGruber**

### CODE REVIEW

**Comment by QuentinGruber** [comment]
based on https://github.com/QuentinGruber/h1z1-server/pull/1849

**Comment by QuentinGruber** [comment]
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/QuentinGruber/h1z1-server/1856?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

Current dependencies on/for this PR:
* `dev`
  * **PR #1850** <a href="https://app.graphite.dev/github/pr/QuentinGruber/h1z1-server/1850?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
    * **PR #1851** <a href="https://app.graphite.dev/github/pr/QuentinGruber/h1z1-server/1851?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
      * **PR #1852** <a href="https://app.graphite.dev/github/pr/QuentinGruber/h1z1-server/1852?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
        * **PR #1853** <a href="https://app.graphite.dev/github/pr/QuentinGruber/h1z1-server/1853?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
          * **PR #1854** <a href="https://app.graphite.dev/github/pr/QuentinGruber/h1z1-server/1854?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
            * **PR #1855** <a href="https://app.graphite.dev/github/pr/QuentinGruber/h1z1-server/1855?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
              * **PR #1856** <a href="https://app.graphite.dev/github/pr/QuentinGruber/h1z1-server/1856?utm_source=stack-comment-icon" target="_blank">Graphite</a>  👈

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/QuentinGruber/h1z1-server/1856?utm_source=stack-comment).

**Comment by Nikolassparrow** [comment]
Players should also spawn with a Jagged skinning knife so they can harvest animals 👍

**Comment by QuentinGruber** [comment]
> Players should also spawn with a Jagged skinning knife so they can harvest animals 👍

true add this to your harvest PR :cowboy_hat_face:

File: data/2016/dataSources/ServerItemDefinitions.json [hunk 1]
```diff
@@ -143955,7 +143957,7 @@
     },
     "3450": {
         "NAME": "Reaper",
-        "ID": 3450,
+        "ID": 1899,
```
Comment (Z1Meme) [inline data/2016/dataSources/ServerItemDefinitions.json:143960]: what is the point of this?

File: data/2016/dataSources/ServerItemDefinitions.json [hunk 2]
```diff
@@ -12285,7 +12286,7 @@
     },
     "1373": {
         "NAME": "Sniper Rifle",
-        "ID": 1373,
+        "ID": 1899,
```
Comment (Z1Meme) [inline data/2016/dataSources/ServerItemDefinitions.json:12289]: This makes no sense?

**Review by Z1Meme — CHANGES_REQUESTED** [review]

**Pull request closed without merging**